gslides_api/pptx/chart_renderer.py

```
@@ -0,0 +1,426 @@
+    except Exception as e:
+        logger.warning(f"Failed to render chart element: {e}")
+        return None
+    finally:
+        # Clean up temp file
+        if temp_path and os.path.exists(temp_path):
+            os.unlink(temp_path)
+
+
+@log_time
+def render_pptx_chart_to_image(
+    pptx_element: "GraphicFrame",
```

Ruff (0.15.6)

[error] Undefined name `GraphicFrame`

(F821)